macos_elevation_fix.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Does this need to run under bash?
iirc macos runs under zsh by default
``#!/bin/sh`` might be sufficient

Reply:
👍